CONTRIBUTING.md

```diff
@@ -53,7 +53,22 @@ about Git.
   - Create pull request
 
 The title of the pull request should be prefixed by the component or area that
-the pull request affects. Examples:
+the pull request affects. Valid areas as:
+
+  - *Consensus* for changes to consensus critical code
+  - *Docs* for changes to the documentation
+  - *Qt* for changes to bitcoin-qt
+  - *Mining* for changes to the mining code
+  - *Net* or *P2P* for changes to the peer-to-peer network code
+  - *Refactor* for code refactors
```

Comment:
This is still prefixed with a module. E.g. [qt] Wallet refactor

The prefix is imo just an indication which folder or file you are touching in that pull.